src/pages/TaskList/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { FaRegTrashAlt } from 'react-icons/fa';
+import { TbClipboardText } from 'react-icons/tb';
+import { Link } from 'react-router-dom';
+import {
+  Titulo,
+  Container,
+  Todo,
+  ContainerTodo,
+  EmptyList,
+  BtnCreateNewTask,
+} from './styles';
+
+import { ITasksProps } from 'types/tasks';
+
+interface TaskListProps {
+  tasks: ITasksProps[];
+  onDelete: (taskId: string) => void;
+  onComplete: (taskId: string) => void;
+}
+
+export const TasksList = ({ tasks, onDelete, onComplete }: TaskListProps) => {
+  return (
+    <>
+      <Titulo>Lista de Tarefas</Titulo>
+      <Container>
+        <ContainerTodo>
+          {tasks.map((t) => (
+            <Todo isCompleted={t.isCompleted} key={t.id}>
+              <div>
+                <h3>Título:</h3>
+                <p> {t.title} </p>
+                <input type="checkbox" onClick={() => onComplete(t.id)} />
+              </div>
+              <div>
+                <h3>Descrição:</h3>
+                <p> {t.description}</p>
+              </div>
+              <button onClick={() => onDelete(t.id)} title="Excluir tarefa">
+                <FaRegTrashAlt />
+              </button>
+            </Todo>
+          ))}
+          {tasks.length <= 0 && (
```

Comment:
outra opção válida de comparação nesse caso:
```suggestion
          {!tasks.length && (
```